app/assets/styles/user.scss

```diff
@@ -1,3 +1,49 @@
+$accent-color: #2C9FA3;
+$accent-color-light: #2DBDC2;
+$subheader-color: #AAAAAA;
+$font-choice: 'Nunito Sans', sans-serif;
+$transform: translate(-50%,0%);
+
 .editor-user {
-    word-wrap: break-word;
+  word-wrap: break-word;
+}
+
+#about, #suggest-resource, #contact, #login, #reset-password, #health, #rights {
```

Comment:
can you add a general class instead and style that one class and add it to the divs with these ids?